### ISSUE TITLE
Extend psf_grid functionality to WFIRST WFI class

### ISSUE THREAD

**Issue by mperrin** [spacetelescope/webbpsf#290]
As discussed in Slack with @shanosborne - 

Right now `psf_grid` is defined on `JWInstrument` so it won't work for the WFIRST `WFI` class. This should be relocated to `SpaceTelescopeInstrument` or else a comparable implementation added to `WFIRSTInstrument` to enable the creation of PSF grids for WFI with the same programming interface. 

@shanosborne to lead modification of `psf_grid`, @robelgeda to support with any necessary knowledge of WFIRST code, please. Seems like this ought to be straightforward.

**Comment by mperrin** [comment]
@shanosborne, @ojustino I think this may be complete? Can this issue be closed?

**Comment by shanosborne** [comment]
Yep, looks like the PR back in 2019 fixed this, and I just used the wrong wording so it did auto-close the issue when the PR merged.
